h/feeds/rss.py

```diff
@@ -73,8 +71,7 @@ def feed_from_annotations(annotations, annotation_url, rss_url, html_url,
             for annotation in annotations]
     }
 
-    if annotations:
-        feed['pubDate'] = parser.parse(annotations[0]['updated']).strftime(
-            '%a, %d %b %Y %H:%M:%S %Z')
+    if annotations and annotations[0].updated:
```

Comment:
Ditto. If `annotations` is truthy, then so is `annotations[0]` (unless it can somehow be something other than an annotation), and `.updated` is not nullable. Overly defensive or am I missing something?